### DISCUSSION TITLE
Assigning an email address to the Google Cal (to accept events sent to that email account)

### DISCUSSION THREAD

**Discussion by al2k** [kewisch/gdata-provider#698] [Support] [answered]
Hi there,

The extension works like a charm (many thanks for your hard work putting this together).

My only issue is there isn't a field to assign an email address to the calendar, so Thunderbird doesn't let me accept an invite / meeting and assign it to the Google Calendar, where as a standard Thunderbird calendar has an "Email" field.

Am I missing anything here?

**Comment by al2k** [comment, marked as answer]
I continued searching and found the solution. 

The answer is that you need to go to the Extension Configuration settings (Tools > Settings > Add-ons > Provider for Google Cal > click the wrench icon > tick "Enable email scheduling".

Then the option is available as usual and it works perfectly.

**Reply by kewisch** [reply]
Thanks you, glad you've found out how to fix it!

**Comment by timstoop** [comment]
This seems to only work for the personal calendar, but not for any additional calendars that are shared with my account? I can see their content and add events, but when I respond to an event in email, I only get the option to add it to my Google Personal calendar, not to another one. Is there a fix for that as well?
